templates/base.html

```diff
@@ -9,6 +9,18 @@
 </head>
 
 <body>
+    <div>
+        {% if request.user.is_authenticated %}
+        <h3><a href="{% url 'tweets:home' %}">Twitter Clone</a></h3>
+        <a href="{% url 'accounts:logout'%}">ログアウトする</a>
+        <a href="{% url 'accounts:user_profile' request.user.username %}"><button type="button">{{ user.username }}</button></a>
```

Comment:
あくまでここは自分のアカウントのprofileに行くリンクなので以下の方が良いでしょう。
というのもprofile画面では、{{user}}に訪問先のユーザーが格納されてしまうからです。templateでは、ログインユーザーを指す時は、request.userが安全でしょう
```suggestion
        <a href="{% url 'accounts:user_profile' request.user.username %}"><button type="button">{{ request.user.username }}</button></a>
```